docs/common/craft-parts/reference/plugins/maven_plugin.rst

```diff
@@ -32,16 +45,41 @@ maven-parameters
 
 Used to add additional parameters to the ``mvn package`` command line.
 
+maven-use-mvnw
+~~~~~~~~~~~~~~
+**Type:** boolean
+
+Used to determine whether the build should use the project provided Maven
+wrapper (located at ``<project-root>/mvnw``) executable file. The command ``mvn
+package`` is replaced with ``./mvnw package`` command.
+
 
 Environment variables
 ---------------------
 
-This plugin reads the ``http_proxy`` and ``https_proxy`` variables from the environment
-to configure Maven proxy access. A comma-separated list of hosts that should not be
-accessed via proxy is read from the ``no_proxy`` environment variable.
+Environment variables can be specified to modify behavior of the build. Three proxy related case
+insensitive environment variables are treated specially.
+``http_proxy``, ``https_proxy`` and ``no_proxy``.
+
+For a list of environment variables used to configure Maven, please refer to
+`Configuring Apache Maven <https://maven.apache.org/configure.html>`_.
+
+http_proxy
+~~~~~~~~~~
+
+URL to proxy http request to. The value is mapped to the settings file (.parts/.m2/settings.xml)
+under proxy element.
+
+https_proxy
+~~~~~~~~~~~
+
+URL to proxy https request to. The value is mapped to the settings file (.parts/.m2/settings.xml)
+under proxy element.
```

Comment:
```suggestion
URL to proxy HTTP request to. The value is mapped to the settings file
(``.parts/.m2/settings.xml``) under the proxy element.

https_proxy
~~~~~~~~~~~

URL to proxy HTTPS request to. The value is mapped to the settings file
(``.parts/.m2/settings.xml``) under the proxy element.
```

Reply:
Thank you for the improvements, updated!